### ISSUE TITLE
Need new release for Carthage support

### ISSUE THREAD

**Issue by mark-anders** [logkit/logkit#22]
Though the merge for #20 might have fixed the issue, Carthage only picks up released builds so it's still picking up 2.3.0.

Can you create a new release so it fully works with Carthage?

**Comment by justinpawela** [comment]
Yep just making sure the new Carthage build testing works.

**Comment by mark-anders** [comment]
ok, super!  Just wasn't sure if you knew about having to do the release...

**Comment by justinpawela** [comment]
...aaaaaaaaand... 2.3.1 has passed its tests and been tagged!

**Comment by mark-anders** [comment]
Don't know whether to laugh or cry... :-)

SO... I thought I'd check it out... just by doing a carthage update with the following in my Cartfile:

`github "logkit/logkit" "master`

This is before you cut 2.3.1

I then did carthage update and got: 

```
*** Building scheme "LogKit tvOS" in LogKit.xcodeproj
** CLEAN FAILED **


The following build commands failed:
    Check dependencies
(1 failure)
** BUILD FAILED **


The following build commands failed:
    Check dependencies
(1 failure)
A shell task failed with exit code 65:
** CLEAN FAILED **


The following build commands failed:
    Check dependencies
(1 failure)
** BUILD FAILED **


The following build commands failed:
    Check dependencies
(1 failure)
```

All other targets (iOS, watchOS, and iOS) work fine.

**Comment by justinpawela** [comment]
Hmmmmm.... I just locally tried the following four Cartfile varieties and had no issues building:

```
github "logkit/logkit" "master"
```

```
github "logkit/logkit" ~> 2.3
```

```
github "logkit/logkit" == 2.3.1
```

```
github "logkit/logkit"
```

Can you post what your LogKit line looks like in your Cartfile?

**Comment by mark-anders** [comment]
I've tried 2 cart files, both with the same result:

`github "logkit/logkit" "master"`

and 

`github "logkit/logkit" ~> 2.3.1`

Both gave me the error.

One thing that just occurred to me: I do not build for tvOS at all.  Is it possible I need to install something to do so?  Or is just having XCode 7.2 enough?

**Comment by mark-anders** [comment]
I found the problem.  I needed to install an Apple TV simulator.  Once  I did that, everything worked fine.

Sorry for the false alarm!

**Comment by justinpawela** [comment]
Okay, good to hear!

Did you remove the tvOS simulator manually? I though that was part of a standard Xcode install, but I'm not sure.

Also, just FYI, I've noted that Carthage will let you select which platforms to build for. So, for instance, if you were only interested in OS X and iOS, you could run:

``` bash
carthage update --platform Mac,iOS
```

`carthage help update` states:

``` text
[--platform all]
    the platforms to build for (one of ‘all’, ‘Mac’, ‘iOS’, ‘watchOS’, 'tvOS', or comma-separated values of the formers except for ‘all’)
```

**Comment by mark-anders** [comment]
I very might well have deleted the things I don't use.  I have some other utilities like Core Data Pro that will present a list of all simulators installed and the list gets long.  Since I'm not building for tvOS (or some old iOS) I just remove them.

I know all about the build for platform as builds take a long time and so I'll do `--platform iOS`. I only did the build all as a way to see if the change I had made worked! :-)  I was pretty shocked when it failed.  Glad to see that it was just an issue with my machine configuration.
